listView.php

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="">
+
+
+<head>
```

Comment:
This <body> might not be required as we will be having a separate BoardView html page to view the results from this listView page

Reply:
Yup, I just added this as a placeholder - can be removed once the BoardView page is finalized.